packages/@dcl/inspector/src/lib/babylon/decentraland/EcsEntity.ts

```diff
@@ -244,22 +248,22 @@ async function validateEntityIsOutsideLayout(entity: EcsEntity) {
 
 function updateMeshBoundingBoxVisibility(entity: EcsEntity, mesh: BABYLON.AbstractMesh) {
   const scene = mesh.getScene()
+  if (scene.isLoading) return
```

Comment:
Condition added to avoid showing bounding boxes while the scene is still loading assets